src/cli/validators.py

```diff
@@ -0,0 +1,34 @@
+import platform
+import os
+import json
+from pathlib import Path
+from typing import Tuple, Any
+from jsonschema import Draft7Validator, RefResolver, draft7_format_checker
+from jsonschema.exceptions import ValidationError
+from cli.utils import read_json_file
+
+
+def validate_json_string(instance_path: Path) -> Tuple[bool, Any]:
```

Comment:
validate_json_string seems not a correct name. validate_json or validate_json_file.

When you feed a string which contains json, validate_json_string is more appropriate.